### PR TITLE
#187 - Default HTTP version in RequestLine

### LINKED ISSUE #187: Default HTTP version in `RequestLine`
`RequestLine` requires HTTP version as one of it's constructor parameters. It's very rare to have version other then `HTTP/1.1`, yet it has to specified every time.

### PR DESCRIPTION
**Pull request by olegmoz**
Closes #187 
Default HTTP version in `RequestLine` is `HTTP/1.1` and constructor accepts `RqMethod` as method argument.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#188](https://github.com/artipie/http/issues/188) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/c041c1c7-bd60-4059-b47a-004e49487474); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/http/rq/RequestLineTest.java [hunk 1]
```diff
@@ -41,4 +41,12 @@ public void reqLineStringIsCorrect() {
             Matchers.equalTo("GET /pub/WWW/TheProject.html HTTP/1.1\r\n")
         );
     }
+
+    @Test
+    public void defaultVersion() {
```
Comment (Vatavuk) [inline src/test/java/com/artipie/http/rq/RequestLineTest.java:46]: @olegmoz expand the test name a little bit to make it more clear

Comment (olegmoz) [reply src/test/java/com/artipie/http/rq/RequestLineTest.java]: @Vatavuk thanks, made it more clear

**Comment by 0crat** [comment]
Job was finished in 22 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR, thanks!

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by g4s8** [comment]
@0crat quality acceptable

**Pull request merged**